modules/ocf_syslog/manifests/init.pp

```diff
@@ -8,6 +8,24 @@
     ],
   }
 
+  ocf::firewall::firewall46 {
+    '100 allow syslog on TCP':
+      opts => {
+        chain  => 'PUPPET-INPUT',
+        proto  => 'tcp',
+        dport  => 514,
+        action => 'accept',
+      };
+
+    '101 allow syslog on UDP':
```

Comment:
I think we can combine this rule with the previous one using `proto => ['tcp', 'udp']`